pkg/monitor/intervalcreation/node_test.go

```diff
@@ -10,6 +11,191 @@ import (
 	monitorserialization "github.com/openshift/origin/pkg/monitor/serialization"
 )
 
+func TestMonitorApiIntervals(t *testing.T) {
+
+	testcase := []struct {
+		name    string
+		logLine string
+		want    monitorapi.EventInterval
+	}{
+		{
+			name:    "status",
+			logLine: `Sep 27 08:59:59.857303 ci-op-747jjqn3-b3af3-f45pk-worker-centralus2-bdp5s kubenswrapper[2397]: I0927 08:59:59.850662    2397 status_manager.go:667] "Failed to get status for pod" podUID=a1947638-25c2-4fd8-b3c8-4dbaa666bc61 pod="openshift-monitoring/prometheus-k8s-0" err="Get \"https://api-int.ci-op-747jjqn3-b3af3.ci2.azure.devcluster.openshift.com:6443/api/v1/namespaces/openshift-monitoring/pods/prometheus-k8s-0\": http2: client connection lost"`,
+			want: monitorapi.EventInterval{
+				Condition: monitorapi.Condition{
+					Level:   0,
+					Locator: "ns/openshift-monitoring pod/prometheus-k8s-0 uid/a1947638-25c2-4fd8-b3c8-4dbaa666bc61 container/",
+					Message: "reason/HttpClientConnectionLost Get \"https://api-int.ci-op-747jjqn3-b3af3.ci2.azure.devcluster.openshift.com:6443/api/v1/namespaces/openshift-monitoring/pods/prometheus-k8s-0\": http2: client connection lost",
+				},
+				From: time.UnixMicro(1664269199857303).In(time.UTC),
+				To:   time.UnixMicro(1664269199857303).In(time.UTC),
+			},
+		},
+		{
+			name:    "reflector",
+			logLine: `Sep 27 08:59:59.853216 ci-op-747jjqn3-b3af3-f45pk-worker-centralus2-bdp5s kubenswrapper[2397]: W0927 08:59:59.849136    2397 reflector.go:347] object-"openshift-monitoring"/"prometheus-adapter-7m6srg4dfreoi": watch of *v1.Secret ended with: an error on the server ("unable to decode an event from the watch stream: http2: client connection lost") has prevented the request from succeeding`,
+			want: monitorapi.EventInterval{
+				Condition: monitorapi.Condition{
+					Level:   0,
+					Locator: "ns/openshift-monitoring pod/prometheus-adapter-7m6srg4dfreoi uid/ container/",
+					Message: "reason/HttpClientConnectionLost unable to decode an event from the watch stream: http2: client connection lost",
+				},
+				From: time.UnixMicro(1664269199853216).In(time.UTC),
+				To:   time.UnixMicro(1664269199853216).In(time.UTC),
+			},
+		},
+		{
+			name:    "kubelet",
+			logLine: `Sep 27 08:59:59.853216 ci-op-747jjqn3-b3af3-f45pk-worker-centralus2-bdp5s kubenswrapper[2397]: E0927 08:59:59.849143    2397 kubelet_node_status.go:487] "Error updating node status, will retry" err="error getting node \"ci-op-747jjqn3-b3af3-f45pk-worker-centralus2-bdp5s\": Get \"https://api-int.ci-op-747jjqn3-b3af3.ci2.azure.devcluster.openshift.com:6443/api/v1/nodes/ci-op-747jjqn3-b3af3-f45pk-worker-centralus2-bdp5s?timeout=10s\": http2: client connection lost"`,
+			want: monitorapi.EventInterval{
+				Condition: monitorapi.Condition{
+					Level:   0,
+					Locator: "ci-op-747jjqn3-b3af3-f45pk-worker-centralus2-bdp5s",
```

Comment:
Should this have more locatorish syntax? Is that a node name? maybe node/ci-op...